functions/interface.php

```diff
@@ -107,7 +106,7 @@ function admin_notices() {
 
 				<?php foreach ( $notices['server_config'] as $notice ) : ?>
 
-					<li><?php echo wp_kses_data( $notice ); ?></li>
+					<li><?php echo print_whitelist_html( $notice ); ?></li>
```

Comment:
I think it would be fine to just whitelist some common HTML tags here. `strong`, `b`, `i`, `em`, `code` would probably cover everything for now. Can always add others as needed.